packages/visual-engine/src/buildItemPreview.ts

```diff
@@ -14,6 +14,7 @@ const adaptExample = (examples: string[], { color, bold }: { color?: string; bol
 
   return examples
     .map((item: string) => item.replace(/<span>/g, `<span style="${style}">`)) //
+    .map((item: string) => `<span>${item}</span>`)
```

Comment:
Why do we need to wrap in another `span` ?

Reply:
because we need to wrap words to display it in more than one line in one piece

Reply:
see here https://github.com/ContentSquare/readapt/pull/25/files#diff-fbcc432f378e3d7601bb1040f296f858a1e17c191b14f42fe38bd2afd4d6b6f6R111